templates/layout.html

```diff
@@ -16,7 +16,7 @@
     <div class="container-fluid">
       <div class="row justify-content-md-center">
         <div class="col-9">
-          <nav class="navbar navbar-expand-xl navbar-dark">
+          <nav class="navbar navbar-expand-md navbar-dark" name="top">
```

Comment:
one last question, why is this changed to `md`? I had it at `xl` because there are too many items in the nav and they don't look good when they're expanded with that viewport size.